src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
It looks like you are trying to link a SCSS file directly in your HTML. Browsers do not understand SCSS files. You should compile your SCSS into CSS and link the CSS file instead.